persistence/src/test/scala/hmda/persistence/processing/HmdaFileValidatorSpec.scala

```diff
@@ -104,20 +107,13 @@ class HmdaFileValidatorSpec extends ActorSpec with BeforeAndAfterEach with HmdaF
       probe.expectMsg(HmdaFileValidationState(
         Some(ts),
         lars,
-        List(
-          SyntacticalValidationError("38800009923", "S025", true),
-          SyntacticalValidationError("38800009923", "S011", true)
-        ),
+        List(),
```

Comment:
The LARs being sent to the `HmdaFileValidator` haven't changed, correct?  Why are there fewer errors being returned in this test?

Reply:
Right. When I introduced this PR I mentioned that I had to change a test to make it work. This is it. I'm not sure why this change happens, though I think the API is doing the right thing when running the project.